src/archivist/services/documentTypes.json

```diff
@@ -1,191 +1,198 @@
+      "object": "human rights"
+    }
+  },
+  "In-App Purchases Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
+      "object": "in-app and other virtual goods purchase usage"
+    }
+  },
+  "Review Guidelines": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "developer",
+      "object": "review process of provided content"
+    }
+  },
+  "Brand Guidelines": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "developer",
+      "object": "usage of the service provider’s brand"
+    }
+  },
+  "Quality Guidelines": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
+      "object": "quality of content produced by means of the service"
+    }
+  },
+  "Data Controller Agreement": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "data controllers (in the sense of GDPR)",
+      "object": "end user personal data"
+    }
+  },
+  "Data Processor Agreement": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "data controllers (in the sense of GDPR)",
+      "object": "status of data processor (in the sense of GDPR) for the service provider"
+    }
+  },
+  "User Consent Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "data controllers (in the sense of GDPR)",
+      "object": "user consent collection and storage"
+    }
+  },
+  "Closed Captioning Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
+      "object": "closed captioning on streamed content"
+    }
+  },
+  "Seller Warranty": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
+      "object": "protection of end users that act as sellers"
+    }
+  },
+  "Single Sign-On Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
+      "object": "connecting user account to other services"
+    }
+  },
+  "Vulnerability Disclosure Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "vulnerability reporter",
+      "object": "how to report a security vulnerability or issue"
+    }
+  },
+  "Live Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
+      "object": "end user’s live service usage"
+    }
+  },
+  "Ad Publishing Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
+      "object": "acceptable and unacceptable usage"
+    }
+  },
+  "Complaints Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "anyone",
+      "object": "how generic complaints will be handled"
+    }
+  },
+  "Transport Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
+      "object": "end user’s transport service usage"
```

Comment:
Based on the [ICAO’s definition](https://www.icao.int/sustainability/Documents/ConsumerInterests_B.pdf):

> Both conditions of carriage/contract spell out the various benefits and limitations associated with the air transportation being provided. These benefits and limitations, along with the price, constitute a “contract for carriage” between the air carrier and the user.

(this could probably be improved by being made a bit shorter)

```suggestion
      "object": "benefits and limitations associated with the transportation being provided"
```

Reply:
👍